arch/arm64/configs/bcm2711_defconfig

```diff
@@ -790,7 +790,9 @@ CONFIG_DRM_LOAD_EDID_FIRMWARE=y
 CONFIG_DRM_UDL=m
 CONFIG_DRM_PANEL_SIMPLE=m
 CONFIG_DRM_PANEL_RASPBERRYPI_TOUCHSCREEN=m
+CONFIG_DRM_V3D=m
 CONFIG_DRM_VC4=m
+
```

Comment:
There should be no blank lines. Ideally, use `make savedefconfig` to regenerate the defconfig files.